build-tools/create-pkg/create-pkg.targets

```diff
@@ -0,0 +1,136 @@
+    />
+    <MakeDir Directories="$(MonoFrameworkExternalDir)\xbuild-frameworks" />
+    <Exec WorkingDirectory="$(MonoFrameworkExternalDir)\xbuild-frameworks"
+        Command="ln -fs &quot;../../../Xamarin.Android.framework/Versions/$(XAVersion)/lib/xamarin.android/xbuild-frameworks/MonoAndroid&quot; ."
+    />
+    <MakeDir Directories="$(MonoFrameworkExternalDir)\monodoc" />
+    <Exec WorkingDirectory="$(MonoFrameworkExternalDir)\monodoc"
+        Command="ln -fs &quot;../../../Xamarin.Android.framework/Versions/$(XAVersion)/lib/monodoc/MonoAndroid-docs.source&quot; ."
+    />
+    <Exec WorkingDirectory="$(MonoFrameworkExternalDir)\monodoc"
+        Command="ln -fs &quot;../../../Xamarin.Android.framework/Versions/$(XAVersion)/lib/monodoc/MonoAndroid-lib.tree&quot; ."
+    />
+    <Exec WorkingDirectory="$(MonoFrameworkExternalDir)\monodoc"
+        Command="ln -fs &quot;../../../Xamarin.Android.framework/Versions/$(XAVersion)/lib/monodoc/MonoAndroid-lib.zip&quot; ."
+    />
+  </Target>
+  <Target Name="_FinalizePayload"
+      DependsOnTargets="_CreateSymbolicLinks">
+    <ReplaceFileContents
+        SourceFile="distribution.xml.in"
+        DestinationFile="$(PkgDistributionDestination)"
+        Replacements="@PACKAGE_TITLE@=Xamarin.Android $(ProductVersion)"
+    />
+    <Exec
+        WorkingDirectory="$(XamarinAndroidSourcePath)"
+        ConsoleToMSBuild="true"
+        Command="git log --no-color --first-parent -n1 --pretty=format:%ct">
+      <Output TaskParameter="ConsoleOutput" PropertyName="UpdateInfoVersion" />
+    </Exec>
+    <WriteLinesToFile
+        File="$(XAFrameworkDir)\updateinfo"
+        Lines="$(UpdateInfoGuid) $(UpdateInfoVersion)"
+        Overwrite="true"
+    />
+    <MakeDir Directories="$(PkgLicenseDestinationEn)" />
+    <Copy
+        SourceFiles="$(PkgLicenseSrcEn)"
+        DestinationFiles="$(PkgLicenseDestinationEn)\License"
+    />
+  </Target>
+  <Target Name="CreatePkg"
+      Condition=" '$(HostOS)' == 'Darwin' "
+      DependsOnTargets="_FinalizePayload">
+    <MakeDir Directories="$(PkgOutputPath)"/>
+    <ItemGroup>
+      <PkgBuildArgs Include="--root &quot;$(PayloadDir)&quot;" />
+      <PkgBuildArgs Include="--identifier com.xamarin.android.pkg" />
+      <PkgBuildArgs Include="--version $(XAVersion)"/>
+      <PkgBuildArgs Include="--install-location &quot;$(PkgInstallDir)&quot; "/>
+      <PkgBuildArgs Include="--scripts &quot;$(PkgScriptsDir)&quot; "/>
+      <PkgBuildArgs Include="&quot;$(PkgOutputPath)/xamarin.android.pkg&quot; "/>
+    </ItemGroup>
+    <Exec Command="pkgbuild @(PkgBuildArgs, ' ')" />
+    <ItemGroup>
+      <ProductBuildArgs Include="--resources &quot;$(PkgResourcesPath)&quot;" />
+      <ProductBuildArgs Include="--distribution &quot;$(PkgDistributionDestination)&quot;" />
+      <ProductBuildArgs Include="--package-path &quot;$(PkgOutputPath)&quot;" />
+      <ProductBuildArgs Include="&quot;$(PkgProductOutputPath)/$(PkgNamePrefix)-$(XAVersion).pkg&quot;" />
```

Comment:
Could this follow the same "naming convention" as the `.vsix` files, e.g. see: https://jenkins.mono-project.com/job/xamarin-android-pr-pipeline-release/103/Azure/

  * `Xamarin.Android.Sdk-OSS-9.2.99.63_HEAD_4e1434c.vsix`
  * `xamarin.android-oss-9.2.99.63.pkg`

The `.pkg` file is missing the branch it came from (which we really should figure out at some point why it's always `HEAD` and not *the actual branch name*) and the commit.

This is because multiple different PRs can plausibly result in the same version; the version, in-and-of-itself, is insufficient for "unique-ness".

The `.vsix` filename is also overridden in the `monodroid` repo, removing the branch + commit information, as builds from the commercial machines *are* assumed to have unique version numbers (which will *also* be bogus for PR builds, but there's fewer of them...).

Reply:
I've updated this to use the exact same naming convention for both OSS .vsix and .pkg files. 
These files should now be named as such:
* `Xamarin.Android.Sdk-OSS-9.2.99.59_create-installers_1a7c188.pkg`
* `Xamarin.Android.Sdk-OSS-9.2.99.59_create-installers_1a7c188.vsix`

I've also got a lead on the GitBranch issue and will look to address that in a separate PR.